### PR TITLE
Add Athena Usage Plugin

### PR DESCRIPTION
**Pull request by MikolajBalcerek**
**Add Athena Usage Plugin**

> Create a dashboard on discreetly with monthly stats who uses how much data
> 
> discreETLy tab have with tiles with information for:
> total monthly Athena usage
> total weekly Athena usage
> total daily Athena usage

https://wikia-inc.atlassian.net/browse/DE-4158

### CODE REVIEW

**Comment by MikolajBalcerek** [comment]
For now this is a naive implementation (no caching of queries)

Things to implement:
- All users summary
- Caching/smart DB implementation, which only queries the dynamodb for the last missing day/24h
- Tests

File: dashboard/plugins/athena_usage/query_dao.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+    def get_finished_queries_for_days_back(self, timespan_days: int) -> List[AthenaQuery]:
+        """
+        This gets queries made from days: int back to today
+        :parameter timespan_days: int number of days to go back
+        :return: list of AthenaQueries containing the request made in that timespan
+        """
+
+        # prepare a list of str dates of days going from now to number timespan_days: int back
+        days_back_dates_str = [datetime.strftime(datetime.now() - timedelta(days=days_back), DATE_FORMAT) for days_back in range(0, timespan_days + 1)]
```
Comment (szczeles) [inline dashboard/plugins/athena_usage/query_dao.py]: You can replace `range(0, X)` with `range(X)`

**Review by szczeles — APPROVED** [review]

**Comment by MikolajBalcerek** [comment]
All users summary was implemented alongside in-memory-caching.
In-memory-caching was suggested due to being simpler, but it comes with a serious disadvantage of being separate for each of the workers. Tomasz said its poor performance wasn't important.

**Comment by szczeles** [comment]
👍

**Pull request merged**